x/supplier/types/message_stake_supplier_test.go

```diff
@@ -36,61 +41,141 @@ func TestMsgStakeSupplier_ValidateBasic(t *testing.T) {
 	}{
 		// address related tests
 		{
-			desc: "invalid address - nil stake",
+			desc: "valid same owner and operator address",
 			msg: MsgStakeSupplier{
-				Address: "invalid_address",
-				// Stake explicitly omitted
+				Sender:       ownerAddress,
+				OwnerAddress: ownerAddress,
+				Address:      ownerAddress,
+				Stake:        &sdk.Coin{Denom: volatile.DenomuPOKT, Amount: math.NewInt(100)},
+				Services:     defaultServicesList,
+			},
+		},
+		{
+			desc: "valid different owner and operator address",
```

Comment:
Do you have one where the sender/signer is neither the operator or owner?

Reply:
Good catch!